tests/testdata/test_addons/package_global_config_multi_input/globalConfig.json

```diff
@@ -387,63 +387,226 @@
+                            "validators": [
+                                {
+                                    "type": "regex",
+                                    "errorMsg": "Input Name must begin with a letter and consist exclusively of alphanumeric characters and underscores.",
+                                    "pattern": "^[a-zA-Z]\\w*$"
+                                },
+                                {
+                                    "type": "string",
+                                    "errorMsg": "Length of input name should be between 1 and 100",
+                                    "minLength": 1,
+                                    "maxLength": 100
+                                }
+                            ],
+                            "field": "name",
+                            "help": "A unique name for the data input.",
+                            "required": true
+                        },
+                        {
+                            "type": "interval",
+                            "field": "interval",
+                            "label": "Interval",
+                            "help": "Time interval of the data input, in seconds.",
+                            "required": true
+                        }
+                    ],
                     "table": {
                         "actions": [
                             "edit",
                             "delete",
                             "clone"
                         ],
-                        "header": [],
-                        "moreInfo": [],
+                        "header": [
+                            {
+                                "label": "Name",
+                                "field": "name"
+                            },
+                            {
+                                "label": "Interval",
+                                "field": "interval"
+                            },
+                            {
+                                "label": "Status",
+                                "field": "disabled"
+                            }
+                        ],
+                        "moreInfo": [
+                            {
+                                "label": "Name",
+                                "field": "name"
+                            },
+                            {
+                                "label": "Interval",
+                                "field": "interval"
+                            },
+                            {
+                                "label": "Status",
+                                "field": "disabled"
+                            }
+                        ],
                         "customRow": {
                             "type": "external",
                             "src": "custom_row"
                         }
-                    }
+                    },
+                    "useInputToggleConfirmation": true
```

Comment:
I am curious why did not you you put this change into the package_global_config_everything

Reply:
ah, ok. It makes sense